.github/workflows/build-python.yml

```diff
@@ -34,18 +34,24 @@ jobs:
           cache: 'pip'
             
       - name: Download workflow artifact
-        uses: dawidd6/action-download-artifact@v2.14.0
+        uses: dawidd6/action-download-artifact@v2.17.0
         with:
           workflow: "build-libs.yml"
           path: ./libs/${{ matrix.os-artifact[1] }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
           name: ${{ matrix.os-artifact[1] }}
-
+      - uses: actions/setup-dotnet@v1
```

Comment:
new fix 2.  the con of this approach is some repetitiveness cross workflows since we have to set up Dotnet in each.  I think this is fine but feel free to roast this